coalib/results/TextPosition.py

```diff
@@ -31,3 +31,46 @@ def line(self):
     @property
     def column(self):
         return self._column
+
+    def __le__(self, other):
+        """
+        Performs comparison between two TextPositions, logically equivalent
+        to self <= other. If the column in a TextPosition is None, consider
```

Comment:
"logically equivalent to self <= other": hm yeah, kinda obvious, that's what `__le__` is for ;D

Maybe just write "Test whether ``self`` is behind the ``other`` Textposition" or so :)